doc/source/examples.rst

```diff
@@ -98,16 +98,17 @@ Linux system or manage services for such things as start-up order, dependency
 checking, and failed service recovery. [1]_
 
 To start a service which requires systemd, configure `molecule.yml` with a
-systemd compliant image, capabilities, volumes, and command as follows.
+systemd compliant image, tmpfs, volumes, and command as follows. [4]_
```

Comment:
Are the people who want `capabilities: SYS_ADMIN` losing something now?

Reply:
Makes sense.  I added back the SYS_ADMIN example and made it clear that it is a privileged container.

I also removed the top paragraph since it seems superfluous now and the first reference since it's 404.  I also added some official references from docker explaining privileged mode and a reference to the capabilities man page further explaining the SYS_ADMIN capability.

Reply:
Thanks!